### PR TITLE
Remove Wire

### PR DESCRIPTION
**Pull request by xDazlD**
## Description

Resolves: #1488 

#### Check List 

- [x] I have read and understand [the contributing guidelines](https://github.com/privacytoolsIO/privacytools.io/blob/master/.github/CONTRIBUTING.md).

- [N/A] The project is [Free Libre](https://en.wikipedia.org/wiki/Free_software) and/or [Open Source](https://en.wikipedia.org/wiki/Open-source_software) Software

* Netlify preview for the mainly edited page: 
https://deploy-preview-1489--privacytools-io.netlify.com/software/real-time-communication/

I'm a first time contributor here and likely did something wrong. Please suggest corrections as needed.

### CODE REVIEW

**Comment by nitrohorse** [comment]
In case we do end up delisting Wire, we’ll also want to remove the image and Worth Mentioning section under https://deploy-preview-1489--privacytools-io.netlify.com/software/real-time-communication/#teamchat.

**Comment by danarel** [comment]
I endorse this pull request.

**Comment by blacklight447-ptio** [comment]
Shouldn't we first figuire the rest out? Removing has quite a dramatic impact, and as long as we dont have a clear picture, people will be wondering what happend

**Comment by danarel** [comment]
I think this article answers almost all of it. Wire isn’t interested in individual privacy. They even removed the word privacy from their slogan. 

In the article, their CEO says their plan all along was to expand enterprise sales and offer security to them and will eventually launch a freemium feature for individuals. They even took down references to the free accounts on the website. 

https://techcrunch.com/2019/11/13/messaging-app-wire-confirms-8-2m-raise-responds-to-privacy-concerns-after-moving-holding-company-to-the-us/

**Comment by blacklight447-ptio** [comment]
I see, that truly is worrying, though i still believe we should put all information on this together to document it, and so we can push out a warning about this today, will you be able to handle this dan?

**Comment by dngray** [comment]
I will be working on this issue https://github.com/privacytoolsIO/privacytools.io/issues/1377 in the coming days so I'll take this into account.

**Comment by xDazlD** [comment]
The pull doesn't need to be accepted right away. It's just here and ready for if the decision was made to go ahead with removal.
When I get a chance I'll take out the things @nitrohorse mentioned.

**Comment by BurungHantu1605** [comment]
I'm in favor of removing it for now. It's fishy and we need to react fast. Doesn't mean that we have to send out warnings or cause any kind of panic at this stage though.

**Comment by blacklight447-ptio** [comment]
@BurungHantu1605 removing is more likely to cause panic rather then a small warning sign

**Comment by dngray** [comment]
> @BurungHantu1605 removing is more likely to cause panic rather then a small warning sign.

I recommend waiting on this as well, approved so just in case we can remove it in a hurry.

This situation has given me the idea to add this point as a disadvantage for centralized systems:

- Ownership, privacy policy and running of the service can change when a single entity controls it.

**Comment by xDazlD** [comment]
Are we able to specify what exactly are we waiting for? I think it would be best if we can agree on what criteria we still need before the removal is put in place.

Do we want to wait for a specific time period to see what happens? Do we want to see what Wire's next statement will be? Do we want to have someone investigate further into Wire's motives behind all of this?

File: _includes/sections/instant-messenger.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <h1 id="im" class="anchor"><a href="#im"><i class="fas fa-link anchor-icon"></i></a> Encrypted Instant Messenger</h1>
 
 <div class="alert alert-warning" role="alert">
-  <strong>If you are currently using an Instant Messenger like Telegram, LINE, Viber, <a href="https://www.eff.org/deeplinks/2016/10/where-whatsapp-went-wrong-effs-four-biggest-security-concerns">WhatsApp</a>, or plain SMS you should pick an alternative here.</strong>
+  <strong>If you are currently using an Instant Messenger like <a href="https://github.com/privacytoolsIO/privacytools.io/issues/1488">Wire</a>, Telegram, LINE, Viber, <a href="https://www.eff.org/deeplinks/2016/10/where-whatsapp-went-wrong-effs-four-biggest-security-concerns">WhatsApp</a>, or plain SMS you should pick an alternative here.</strong>
```
Comment (Mikaela) [inline _includes/sections/instant-messenger.html]: I am not happy with Wire being listed here as they are still open source.

However a blog post should probably be made like with Startpage. @privacytoolsIO/content ?

File: _includes/sections/instant-messenger.html [hunk 2]
```diff
@@ -22,24 +22,6 @@ <h1 id="im" class="anchor"><a href="#im"><i class="fas fa-link anchor-icon"></i>
 ios="https://apps.apple.com/us/app/signal-private-messenger/id874139669"
 %}
 
-
-{% include cardv2.html
-title="Wire"
-image="/assets/img/tools/wire.png"
```
Comment (Mikaela) [inline _includes/sections/instant-messenger.html]: As nitrohorse said, please remove the image.

**Comment by Mikaela** [comment]
Sorry, I forgot that the Real Time Communication page has three sections from different files. To clarify, what I and @nitrohorse wish you to do is:

* [ ] `git rm assets/img/tools/wire.png`
* [ ] Remove the worth mentioning section from `_includes/sections/teamchat.html` (because it would be left empty and bad looking)

*Sorry for putting words into your mouth, nitrohorse*

**Comment by xDazlD** [comment]
Removed the unused image and worth mentioning as @nitrohorse suggested. Also got rid of a mention of a Wire security audit I missed.
As for adding Wire to the do not use list, the reasoning behind that was to provide a link back to the issue that caused its removal. It was there to solve the problem of a user wondering why Wire vanished. Also, while Wire is open source, it has a policy problem. People do care about that policy problem, that's why this pull request is here. Open to others thoughts on what should be done on that though.

File: _includes/sections/instant-messenger.html (hunk 1 above)
Comment (Mikaela) [reply _includes/sections/instant-messenger.html]: This is now the only thing I am not certain about in this PR, but a blog post is coming.

**Pull request merged**